pytorch_lightning/callbacks/progress/base.py

```diff
@@ -159,6 +159,8 @@ def print(self, *args, **kwargs):
 
     def setup(self, trainer: "pl.Trainer", pl_module: "pl.LightningModule", stage: Optional[str] = None) -> None:
         self._trainer = trainer
+        if not trainer.is_global_zero:
```

Comment:
quick ques: this will now disable with `predict` too? since this is not the case on master. was this a bug before?

Reply:
yes. I believe this was a bug before

Reply:
mind update the changelog? with a test?

Reply:
Done!